contractsSol5/KyberNetwork.sol

```diff
@@ -315,48 +294,15 @@ contract KyberNetwork is Withdrawable2, Utils4, IKyberNetwork, ReentrancyGuard {
 
     /// @dev no. of KyberNetworkProxies are capped
     function addKyberProxy(address networkProxy) external onlyAdmin {
-        require(networkProxy != address(0), "proxy 0");
-        require(!kyberProxyContracts[networkProxy], "proxy exists");
-        require(kyberProxyArray.length < MAX_APPROVED_PROXIES, "Max 2 proxy");
-
-        kyberProxyArray.push(networkProxy);
-
-        kyberProxyContracts[networkProxy] = true;
```

Comment:
we need this
Since as part of trade flow.
we dont want to call external contract per trade, due to gas cost.
unless good enough reason
so only array should move to storage contract.

Reply:
change as request